app.py

```diff
@@ -42,7 +43,11 @@ def upload_file():
     if file and allowed_file(file.filename):
         filename = secure_filename(file.filename)
         file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
-        os.system('python ./' + convert_from + '2' + convert_to + '.py')
+        file_in = glob.glob('./uploads/*.json')
+        if (convert_from == "json" and convert_to == "png"):
```

Comment:
Next PR; Let's create a dependent mapping to facilitate this check and make it more modular.
So something like a dict like so: (obviously it doesn't have to be exactly this)
```python
supported_from_to_conversions = {
    "json" : {
        "png": 1 | True,
        "glm": 1 | True,
        ...
    },
   ...
}

try:
    if(supported_from_to_conversions[convert_from][convert_to]):
        ....
except KeyError:
    ....
```